--- libcap-ng-0.7/bindings/python/capng.py.orig	2012-09-14 21:32:13.379634254 +0200
+++ libcap-ng-0.7/bindings/python/capng.py	2012-09-14 21:34:15.389631713 +0200
@@ -103,6 +103,8 @@
 CAP_MAC_ADMIN = _capng.CAP_MAC_ADMIN
 CAP_SYSLOG = _capng.CAP_SYSLOG
 CAP_WAKE_ALARM = _capng.CAP_WAKE_ALARM
+CAP_BLOCK_SUSPEND = _capng.CAP_BLOCK_SUSPEND
+CAP_CONTEXT = _capng.CAP_CONTEXT
 CAP_LAST_CAP = _capng.CAP_LAST_CAP
 CAPNG_DROP = _capng.CAPNG_DROP
 CAPNG_ADD = _capng.CAPNG_ADD
diff -upr libcap-ng-0.6.3./src/captab.h libcap-ng-0.6.3/src/captab.h
--- libcap-ng-0.6.3./src/captab.h	2009-03-16 18:12:32.000000000 +0100
+++ libcap-ng-0.6.3/src/captab.h	2010-04-11 20:14:16.641332609 +0200
@@ -60,4 +60,7 @@ _S(CAP_MAC_OVERRIDE,		"mac_override"		)
 #ifdef CAP_MAC_ADMIN
 _S(CAP_MAC_ADMIN,		"mac_admin"		)
 #endif
+#ifdef CAP_CONTEXT
+_S(CAP_CONTEXT,			"context"		)
+#endif
 
